src/pages/staking/index.js

```diff
@@ -1,422 +1,758 @@
+                    tokens that represents your staked ETH plus your share of
+                    the validator rewards
                   </li>
                   <li>
-                    <Translation id="page-staking-upgrades-li-4" />
+                    Liquidity tokens can be held in your own wallet, used in
+                    DeFi and sold if you decide to exit
+                  </li>
+                </ul>
+              </div>
+              <div
+                style={{
+                  gridArea: "pool-risks",
+                  borderBottom: "1px solid #3335",
+                }}
+              >
+                <h4>Risks</h4>
+                <ul>
+                  <li>Risks vary depending on the method used</li>
+                  <li>
+                    In general, risks consist of a combination of counter-party,
+                    smart contract and execution risk
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "pool-reqs" }}>
+                <h4>Requirements</h4>
+                <ul>
+                  <li>
+                    Lowest ETH requirements, some projects require as little as
+                    0.01 ETH
                   </li>
                   <li>
-                    <Translation id="page-staking-upgrades-li-5" />
+                    Deposit directly from your wallet to different pooled
+                    staking platforms or simply trade for one of the staking
+                    liquidity tokens
                   </li>
                 </ul>
-              </BoxText>
-            </Box>
-          </Column>
-        </Row>
-        <H2>
-          <Translation id="page-staking-benefits" />
-        </H2>
-        <CardContainer>
-          <StyledCard
-            emoji=":evergreen_tree:"
-            title={translateMessageId("page-staking-sustainability", intl)}
-            description={translateMessageId(
-              "page-staking-sustainability-desc",
-              intl
-            )}
-          />
-          <StyledCard
-            emoji=":globe_showing_americas:"
-            title={translateMessageId("page-staking-accessibility", intl)}
-            description={translateMessageId(
-              "page-staking-accessibility-desc",
-              intl
-            )}
-          />
-          <StyledCard
-            emoji=":old_key:"
-            title={translateMessageId("page-staking-sharding", intl)}
-            description={translateMessageId("page-staking-sharding-desc", intl)}
-          >
-            <Link to="/upgrades/shard-chains/">
-              <Translation id="page-staking-more-sharding" />
-            </Link>
-          </StyledCard>
-        </CardContainer>
-      </Content>
-    </Page>
+              </div>
+              <div style={{ gridArea: "pool-cta" }}>
+                <StyledButtonLink
+                  onClick={() => {
```

Comment:
Let's remove these events - this is just a link, we can track this via pageviews.